python/setup.py

```diff
@@ -202,6 +202,18 @@ def get_option_from_sys_argv(option_str):
             r'macosx-[0-9]+\.[0-9]+-(.+)', r'macosx-10.9-\1',
             util.get_platform())
 
+    # https://developer.apple.com/documentation/xcode_release_notes/xcode_10_release_notes
```

Comment:
Not related to hunterization, please remove, see:
- https://docs.hunter.sh/en/latest/creating-new/patch.html

Reply:
I didn't add this, it's from upstream.

Reply:
There is no such change in 3.7.1 tag. At this moment `hunter-3.7.1` branch is a clean 3.7.1 upstream version.